### ISSUE TITLE
Can the typings from this repo be moved over to the actual node-amqp10 repo?

### ISSUE THREAD

**Issue by amarzavery** [typed-contrib/node-amqp10#3]
@michaelinipop, @SomaticIT  - Can the typings from this repo be moved over to the actual node-amqp10 repo?
In that way we can have `"types"` property in package.json and ship typings along with the package.
What do you guys think?

**Comment by iandvt** [comment]
+1 this would be awesome.

**Comment by SomaticIT** [comment]
Of course, this work can be merged inside the core `node-amqp10` project.
Please contact the project owner to know if he is interested in this merge?

Thank

**Comment by lostfields** [comment]
if that isn't possible, why not include it in npm `@types/amqp10` ?

https://www.npmjs.com/search?q=%40types

**Comment by SomaticIT** [comment]
I tried but do not have the rights on the `@types` namespace.
Maybe should we merge with the definitely typed repo which is responsible for the `@types` namespace.
